parent-pom.xml

```diff
@@ -25,7 +25,9 @@
     <avro.version>1.8.1</avro.version>
     <awaitility.version>4.2.0</awaitility.version>
     <awssdk.version>1.12.655</awssdk.version>
-    <azure.storage.version>5.0.0</azure.storage.version>
+    <azure.storage.blob.version>12.26.1</azure.storage.blob.version>
+    <azure.storage.version>8.0.0</azure.storage.version>
```

Comment:
the newest is 8.6.6 - can we use it?

Reply:
Right, I haven't noticed the newer version but it'll be rather no-op anyway